gobblin-runtime/src/main/java/gobblin/runtime/JobState.java

```diff
@@ -78,6 +80,13 @@ public JobState(String jobName, String jobId) {
     this.setId(jobId);
   }
 
+  public JobState(State properties, List<WorkUnitState> previousTaskStates, String jobName, String jobId) {
+    super(properties, previousTaskStates);
+    this.jobName = jobName;
+    this.jobId = jobId;
+    this.setId(jobId);
```

Comment:
Seems `this.jobId` always equals `this.id`. What's the purpose of having `jobId`?